scripts.js

```diff
@@ -1,6 +1,14 @@
 // Containers - use those for interaction with them
 const container1 = document.getElementById('container-1');
 const container2 = document.getElementById('container-2');
+const tileHeight = 40;
```

Comment:
Usually the convention for constants in js is all Upper case with underscore delimiter.
const TILE_HEIGHT = 40